CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Change Log
 ==========
 
+## Unreleased
+- `Synchronizer.progress` now returns Flow<PercentDecimal> instead of Flow<Int>. `PercentDecimal` is type-safe model.
```

Comment:
We should also update the release notes to include the sync status changes, since those were updated in the prior PR for the CompactBlockProcessor refactor.

Reply:
Yes, good idea.